skills/declarative/Calendar/Calendar/dialogs/GetEventsDialog/GetEventsDialog.dialog

```diff
@@ -65,7 +70,8 @@
         {
           "$kind": "Microsoft.IfCondition",
           "$designer": {
-            "id": "isyBmx"
+            "id": "isyBmx",
+            "comment": "The $doSelection property indicates whether to return a list of events, or to require the user to select a specific event in order to continue. When showing a schedule, or search results, $doSelection should be false. If performing an event update or event cancellation, $doSelection should be true."
```

Comment:
I'm wondering if it might be nicer to have the definitions of these kinds of properties up at the top. Going through this dialog step by step I learned all about $doSelection at this node but there are other properties in this dialog and it would be nice to get a whole view of all of them.

On a similar note, there isn't a comment for $turnCount when it is set.

One idea: have the comment at the top contain definitions for referenced properties but then the node where they are manipulated has a comment explaining the logic there. In this case, using the first $doSelection sentence in a top-down comment and then explaining the true/false differences here.

Reply:
I added a top level definition for the doSelection property and updated the comments for the other similar properties across the project